install-git.Rmd

```diff
@@ -61,7 +61,7 @@ If you already have Git for Windows, but it's not the latest version, it's a goo
 Since v2.16, you can [update like so from the command line](https://stackoverflow.com/a/48924212):
```

Comment:
Let's try to also keep the surrounding text up-to-date.

This looks like the right reference to me. If you agree, please update this sentence. We can probably drop "Since v2.16" and swap out the URL.

https://github.com/git-for-windows/git/wiki/FAQ#how-do-i-update-git-for-windows-upon-new-releases

Reply:
Thanks, Jenny, for reviewing the change. I agree. I've updated the sentence and reference as suggested.